src/interfaces/incorporation-application-interface.ts

```diff
@@ -1,5 +1,4 @@
-import { IncorporationAddressIf, ShareClassIF,
-  NameTranslationIF } from '@/interfaces'
+import { IncorporationAddressIf, ShareClassIF, NameTranslationIF } from './'
```

Comment:
The previous code had this interface looking in the parent's interfaces folder for these imports. This is incorrect. This interface should look in THIS folder for any imports.

Any shared interface (or mixin, enum, etc) should be able to get everything it needs from the current folder structure (this repo).

It's slightly different if an interface needs an enum... In this example, the interface should import from "@bcrs-shared-components/enums" so that it gets the versioned one.